samples/csharp_dotnetcore/50.teams-messaging-extensions-search/Resources/connectorCard.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": "https://schema.org/extensions",
+  "@type": "MessageCard",
+  "sections": [
+    {
+      "activityTitle": "Steve Tweeted",
+      "activitySubtitle": "With #MicrosoftTeams",
+      "activityImage": "https://teamsmessagingextensionssearch20210304124622.azurewebsites.net/MSC12_Oscar_002.jpg"
```

Comment:
Can this be tempated? from BaseUrl?

Reply:
Let me check this

Reply:
not sure if templating in message card supported, I've not seen yet  but for sure i've checked it for Adaptive cards which works like a charm